package.json

```diff
@@ -1,29 +1,36 @@
 {
-  "name": "web",
-  "version": "5.0.0",
-  "description": "angular my-uw",
+  "name": "@uportal/uportal-home",
```

Comment:
:question: per #700 could/should this be `@uportal/home`?

Reply:
whoops! oh yeah!